docs/content/preview/yugabyte-voyager/migrate/assess-migration.md

```diff
@@ -33,7 +33,7 @@ When you run an assessment, Voyager collects metadata or metrics from the source
 The recommendations are based on testing using a [RF3](../../../architecture/docdb-replication/replication/#replication-factor) YugabyteDB cluster on instance types with 4GiB memory per core and running v2024.1.
 {{< /warning >}}
 
-Note that for cases where it is not possible to provide database access to the client machine running Voyager, you can gather the metadata from the source database using plain bash/psql scripts provided with Voyager, and then use Voyager to analyze the metadata directly.
+Note that if providing database access to the client machine running Voyager is not possible, you can gather metadata from the source database using the provided bash scripts and then use Voyager to assess the migration. Currently, these scripts require the machine to have Voyager installed due to dependencies (like psql, sqlplus clients, and so on) included with Voyager. Standalone installers are planned to be provided for these dependencies in the future.
```

Comment:
Let's keep this only to
`
Note that if providing database access to the client machine running Voyager is not possible, you can gather metadata from the source database using the provided bash scripts and then use Voyager to assess the migration.
`